ibridges/data_operations.py

```diff
@@ -64,13 +64,15 @@ def upload(
     copy_empty_folders:
         Create respective iRODS collection for empty folders. Default: True.
     options:
-        More options for the upload
+        Python-irodsclient options found in ``irods.keywords``. The following keywords will be
+        ignored since they are set by iBridges:
+        FORCE_FLAG_KW, RESC_NAME_KW, NUM_THREADS_KW, REG_CHKSUM_KW, VERIFY_CHKSUM_KW.
```

Comment:
Can we make the RESC_NAME_KW settable for the advanced users and other irods instances than Yoda?